src/raiden_libs/cli.py

```diff
@@ -60,6 +61,22 @@ def validate_address(
     return to_canonical_address(value)
 
 
+def start_profiler(output_dir: Optional[str]) -> Optional[Any]:
+    """ Start and return a profiler is `output_dir` is not None
+    """
+    if not output_dir:
+        return None
+
+    from raiden.utils.profiling.sampler import SignalSampler, FlameGraphCollector
+
+    os.makedirs(output_dir, exist_ok=True)
+    now = datetime.datetime.now()
+    stack_path = os.path.join(output_dir, f"{now:%Y%m%d_%H%M}_stack.data")
+    stack_stream = open(stack_path, "w")
```

Comment:
I think we should close the handle somewhere

Reply:
That is done by `FlameGraphCollector.stop`.